src/test/ui/suggestions/suggest-mut-method-for-loop.rs

```diff
@@ -0,0 +1,17 @@
+use std::collections::HashMap;
+struct X(usize);
+struct Y {
+    v: u32
+}
+
+fn main() {
+    let mut buzz = HashMap::new();
+    buzz.insert("a", Y { v: 0 });
+
+    for mut t in buzz.values() {
+                  //~^ HELP
```

Comment:
How come this help is empty? This is also the first time I came across `SUGGESTION`.

Reply:
I use this same as `src/test/ui/suggestions/suggest-ref-mut.rs`. In `suggest-ref-mut`, the help seems available to be empty.
The `SUGGESTION` is introduced by https://rustc-dev-guide.rust-lang.org/tests/adding.html#error-levels.